src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -207,22 +132,21 @@ export async function $onEmit(context: EmitContext<NetEmitterOptions>) {
                 )} --project-path ${outputFolder} ${newProjectOption} --clear-output-folder ${
                     options["clear-output-folder"]
                 }${debugFlag}`;
-                console.info(command);
+                logger.verbose(command);
 
                 try {
                     execSync(command, { stdio: "inherit" });
                 } catch (error: any) {
-                    if (error.message) console.log(error.message);
-                    if (error.stderr) console.error(error.stderr);
-                    if (error.stdout) console.log(error.stdout);
-
+                    if (error.message) logger.info(error.message);
+                    if (error.stderr) logger.error(error.stderr);
+                    if (error.stdout) logger.verbose(error.stdout);
                     throw error;
                 }
             }
 
             if (!options["save-inputs"]) {
                 // delete
-                deleteFile(resolvePath(generatedFolder, "cadl.json"));
+                deleteFile(resolvePath(generatedFolder, "cadl2.json"));
```

Comment:
are this file same as ln154 which is still `cadl.json`?

Reply:
consider to predefine this as a const

Reply:
I updated to use constant.